lib/index.js

```diff
@@ -22,4 +22,5 @@ module.exports = {
   squareRoot: require('./squareRoot'),
   sum: require('./sum'),
   xor: require('./xor'),
+  average: require('./average')
```

Comment:
Not a big deal, but try to keep these in alphabetical order moving forward! Thanks!

Reply:
@BlakeGuilloud Thank you so much :)